docs/pages/eas-update/github-actions.mdx

```diff
@@ -92,3 +79,84 @@ Your GitHub Action should be set up now. Whenever a developer merges code into t
 > Some repositories or organizations might need to explicitly enable GitHub Workflows and allow third-party Actions.
 
 </Step>
+
+## Publish previews on pull requests
+
+Another common use case it to create a new update for every pull request. This allows you to test the changes in the pull request on a device before merging the code, and without having to start the project locally. Below are the steps to publish an update every time a pull request is opened:
+
+<Step label="1">
+
+Create a file path named **.github/workflows/preview.yml** at the root of your project.
+
+</Step>
+
+<Step label="2">
+
+Inside **preview.yml**, copy and paste the following snippet:
+
+```yaml
+name: preview
+on: pull_request
+
+jobs:
+  update:
+    name: EAS Update
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Check for EXPO_TOKEN
+        run: |
+          if [ -z "${{ secrets.EXPO_TOKEN }}" ]; then
+            echo "You must provide an EXPO_TOKEN secret linked to this project's Expo account in this repo's secrets. Learn more: https://docs.expo.dev/eas-update/github-actions"
+            exit 1
+          fi
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: yarn
+
+      - name: Setup EAS
+        uses: expo/expo-github-action@v8
+        with:
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Create preview
+        uses: expo/expo-github-action/preview@v8
+        with:
+          command: eas update --auto
+```
+
+Although the code above is similar to the code in the previous example, there are a few differences. In this action, we changed the workflow event (`on`) to run every time a pull request is opened or updated. In the `update` job, we still set up Node, Expo's GitHub Action, and the dependencies using GitHub action's built in cache. But, instead of running `eas update --auto` ourselves, we let the [preview subaction](https://github.com/expo/expo-github-action/tree/main/preview#readme) run it for us. This will add a comment to the pull request with basic information about the update, and a QR code to scan the update.
+
+> Don't forget to add the `permissions` section to the job. This enables the job to add comments to the pull request.
+
+</Step>
+
+<Step label="3">
+
+Finally, just like the previous example, we need to give the script above permission to run by providing an `EXPO_TOKEN` environment variable.
```

Comment:
```suggestion
You must give the script above permission to run by providing an `EXPO_TOKEN` environment variable.
```

Since we're re-listing the steps, we don't have to mention the previous section.